plugin/pkg/seed/validator/admission_test.go

```diff
@@ -98,17 +89,6 @@ var _ = Describe("validator", func() {
 				Expect(err).To(BeForbiddenError())
 			})
 
-			It("should disallow seed deletion because it is still referenced by a backupbucket", func() {
```

Comment:
Can you keep this test case and rather invert it? E.g.:
`should allow seed deletion if it is referenced by BackupBucket (will get cleaned up after Seed deletion)`

Though, it wouldn't really test concrete part of the implementation now, I think it still makes sense to have such tests, that will pop up when someone in the future might want to change the logic. Then they at least have to change the test, but it wouldn't go unnoticed, that there was a concrete reason for doing it the way it is currently implemented.